src/Entity/IncludeCriteria.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace Ups\Entity;
+
+use DOMDocument;
+use DOMElement;
+use DOMNode;
+use Ups\NodeInterface;
+
+class IncludeCriteria implements NodeInterface
+{
+
+    /**
+     * @var ServiceOffering[]
+     */
+    private $serviceOfferingList;
```

Comment:
I think we can make them empty arrays by default, and require the set methods to take only arrays as arguments. Besides that, I think it would be good to add a 'addServiceOffering', etc. which can only take the required object implementation.